tools/dkcm/clerk/clerk.go

```diff
@@ -26,7 +27,7 @@ import (
 
 // Cluster stores a row in the "Cluster" db table
 type Cluster struct {
```

Comment:
The `Cluster` and `Request` struct have a few fields in common. It'll be better to extract these fields to a common struct, something like below:
```
type ClusterConfig struct {
  Zone      string
  Nodes     int
  NodeType  string
}

type Cluster struct {
  ClusterConfig
  ...
}

type Request struct {
  ClusterConfig
  ...
}
```

Then you can use `ClusterConfig` as the parameter of the `CheckAvail` function.

Reply:
That makes sense to me